middlewares/authzCourse.sql

```diff
@@ -1,3 +1,20 @@
+-- BLOCK insert_xc101_viewer
+WITH
+example_course AS (
+    SELECT * FROM pl_courses WHERE (options->'isExampleCourse')::boolean IS TRUE
```

Comment:
Unfortunately this does a sequential scan. Could we add a column to the `questions` table for `is_example_course` and make an index on this, so that we can very quickly find the example course? This would require adding the column and also modifying the sync code to set this new column. I think it would be much cleaner than carrying around this information in the course options.

Reply:
Added here:
a2369d0fd6004eeea4f7346e6622b9220528679e (Add `pl_courses.is_example_course` column)
740f63b01cbf375fcb6aeaa3a84050f38c572855 (Port `options->isExampleCourse` to `is_example_course`)

Reply:
I removed `options->isExampleCourse` entirely. Do I need to leave it as a legacy option? Not sure how deploy/production works.

Reply:
I think it's fine (and better) to completely remove `options.isExampleCourse`.